src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$cell-size: 100px;
+$gap: 1px;
+$month-days: 31;
+$month-min-days: 31;
```

Comment:
Unused var

Reply:
sorry, I didn't notice that I assigned the `$month-min-days` the same value as the `$month-days`